components/dialog/dialog.js

```diff
@@ -71,27 +71,6 @@ class Dialog extends LocalizeStaticMixin(AsyncContainerMixin(DialogMixin(LitElem
 		`];
 	}
 
-	static get resources() {
-		return {
-			'ar': { close: 'إغلاق مربع الحوار هذا' },
-			'da': { close: 'Luk denne dialogboks' },
-			'de': { close: 'Dieses Dialogfeld schließen' },
-			'en': { close: 'Close this dialog' },
-			'es': { close: 'Cerrar este cuadro de diálogo' },
-			'fr': { close: 'Fermer cette boîte de dialogue' },
-			'ja': { close: 'このダイアログを閉じる' },
-			'ko': { close: '이 대화 상자 닫기' },
-			'nl': { close: 'Dit dialoogvenster sluiten' },
-			'pt': { close: 'Fechar esta caixa de diálogo' },
-			'sv': { close: 'Stäng dialogrutan' },
-			'tr': { close: 'Bu iletişim kutusunu kapat' },
-			'tr-tr': { close: 'Bu diyalog kutusunu kapat' },
-			'zh': { close: '关闭此对话框' },
```

Comment:
@dbatiste It seems that we generally assign `zh-cn` translations to `zh` - do you know why there is a separate `close` lang term for `zh` vs `zh-cn`? Similarly with Turkish we generally just have `tr`.

Reply:
I think I copied from elsewhere, so no other particular reason.